hibernate-core/src/main/java/org/hibernate/engine/jdbc/batch/internal/BatchingBatch.java

```diff
@@ -30,8 +30,6 @@ public class BatchingBatch extends AbstractBatchImpl {
 			BatchingBatch.class.getName()
 	);
 
-	// IMPL NOTE : Until HHH-5797 is fixed, there will only be 1 statement in a batch
```

Comment:
I delete the above outdated comment along the way; [HHH-](https://hibernate.atlassian.net/browse/HHH-5797) has been fixed since 5.2.15.